app/decorators/user_decorator.rb

```diff
@@ -73,6 +73,25 @@ def link_with_compact_name(options = {})
     link_to compact_name, profile_path(user.screen_name), options
   end
 
+  def gender
+    return if user.gender.blank?
+    return unless Privacy.for(user, 'GENDER').visible
+    return if user.gender == "NOTSPECIFIED"
+    genders = {
+      "MALE"  => "Férfi",
+      "FEMALE" => "Nő",
+      "UNKNOWN" => "Egyéb"
+    }
+    info_box(genders[user.gender], 'uk-icon-info')
```

Comment:
This mapping is already set in the [application.rb](https://github.com/kir-dev/pek-next/blob/master/config/application.rb#L61) under `config.x.genders`.

What do you think about a solution like
```suggestion

    decorated_gender = Rails.configuration.x.genders[user.gender]
    info_box(decorated_gender, 'uk-icon-info')
```